test/e2e/app-dir/app-routes/app-custom-routes.test.ts

```diff
@@ -13,6 +14,9 @@ const basePath = process.env.BASE_PATH ?? ''
 describe('app-custom-routes', () => {
   const { next, isNextDeploy, isNextDev, isNextStart } = nextTestSetup({
     files: __dirname,
+    dependencies: {
+      '@types/node-fetch': devDependencies['@types/node-fetch'],
```

Comment:
```suggestion
      '@types/node-fetch': 'latest'
```

Reply:
As we pinned repo's node-fetch type as `"@types/node-fetch": "2.6.1"`, it may be potentially flaky if we set it as latest?

Reply:
we can also pin it here, it should be fine as it's not related to the `package.json` in the parent folder. The app is `test/e2e/app-dir/app-routes/`